src/scheduling/appointments/AppointmentDetailForm.tsx

```diff
@@ -134,10 +133,8 @@ const AppointmentDetailForm = (props: Props) => {
               label={t('scheduling.appointment.reason')}
               value={appointment.reason}
               isEditable={isEditable}
-              onChange={
-                (event: React.ChangeEvent<HTMLTextAreaElement>) =>
-                  onFieldChange && onFieldChange('reason', event.currentTarget.value)
-                // eslint-disable-next-line react/jsx-curly-newline
```

Comment:
This ignore is no longer needed since it was conflicting with `prettier/react`. With prettierrc extends updated `eslint-config-prettier` is able to disable the conflicting prettier rule, opting for `jsx-curly-newline` in this case.

Reply:
Ok, remove it!